scss/_navbar.scss

```diff
@@ -65,26 +98,18 @@
 // Used for brand, project, or site names.
 
 .navbar-brand {
```

Comment:
The Boosted mod of the beginning of this definition is incorrect. It should take into account that things are present in Bootstrap and not here. And some things only in Boosted doesn't have their Boosted mod.